src/ProviderRequestApprenticeTraining/SFA.DAS.ProviderRequestApprenticeTraining/Application/Commands/SubmitProviderResponse/SubmitProviderResponseCommand.cs

```diff
@@ -9,9 +9,12 @@ public class SubmitProviderResponseCommand : IRequest<SubmitProviderResponseResp
     {
         public long Ukprn { get; set; }
         public List<Guid> EmployerRequestIds { get; set; } = new List<Guid>();
+        public string ContactName { get; set; }
         public string Email { get; set; }
         public string Phone { get; set; }
         public string Website { get; set; }
         public string CurrentUserEmail { get; set; }
+        public Guid RespondedBy { get; set; }
+        public string CurrentUserFirstName { get; set; }
```

Comment:
This is out of sync as this would have been introduce during the email sending story

Reply:
I added it here because it was mentioned as missing in the PR comments for  https://github.com/SkillsFundingAgency/das-rat-api/pull/24 so i added the column as part of the api changes for 2131 .  Let me know if you want it removed intil we are actually using it for the send email story